.ci/azure/android_arm64.yml

```diff
@@ -122,6 +122,7 @@ jobs:
         -DENABLE_OV_PADDLE_FRONTEND=OFF
         -DBUILD_java_api=ON
         -DBUILD_nvidia_plugin=OFF
+        -DBUILD_custom_operations=OFF
```

Comment:
why do not custom operations work on Android?

Reply:
How is OpenVINO runtime's dependency on TBB solved on Android? The custom operations component isn't introducing a new dependency with TBB, is it?